### ISSUE TITLE
Dot notation broken inside parens.

### ISSUE THREAD

**Issue by gerane** [PowerShell/EditorSyntax#15]
This might be related to #14 

It looks like just having a dot inside of parens breaks the Syntax Highlighting in the latest Insiders Preview 7/27

Sample Code:

``` powershell
if ($Host.name -eq 'ConsoleHost')
{
    if (! (Get-Item ENV:VSCODE_PID -ErrorAction SilentlyContinue))
    {
        ## Set Location
        Set-Location -Path Repo:
    }
}

```

This is how it is by default.

![1syntaxparens](https://cloud.githubusercontent.com/assets/6662208/17176442/5f66e85c-53d2-11e6-882c-2e2c633d37f7.png)

Removing the dot makes everything OK.

![1syntaxparens2](https://cloud.githubusercontent.com/assets/6662208/17176444/611bc992-53d2-11e6-8514-710fd82f1d54.png)

Just having a dot at end of Var breaks syntax highlighting.

![1syntaxparens3](https://cloud.githubusercontent.com/assets/6662208/17176530/c221cbce-53d2-11e6-8980-e587cbb327b8.png)

This one is pretty bad, it breaks a quite a lot of the PowerShell I see. May have to manually load an old syntax.

Off Topic: What syntax highlight method does Github use? they seem to have decent results with PowerShell. Always been better than tmtheme overall.

**Comment by gravejester** [comment]
Yeah, I know what's causing this. It's the part trying to match the property of an object. I'm seeing too many issues with this, so I'm going to disable it entirely and push an update.

**Comment by vors** [comment]
@gerane GitHub uses current SublimeText bundle. https://github.com/github/linguist/tree/master/vendor/grammars

**Comment by omniomi** [comment]
Fixed by #94
